.travis.yml

```diff
@@ -74,6 +74,9 @@ jobs:
       before_script:
         - alpine ./check.py --only-prepare
       script:
+        - if [ "${TRAVIS_EVENT_TYPE}" != "cron" ]; then
+            exit 0;
+          fi        
```

Comment:
Does this work?  I seems to remember that each element of the script list will run .. even if the others fail.  i.e. this is not a script, but a list of scripts.  I could have changed since my last travis work though I guess..

Reply:
The build from my branch push seems to have done the right thing? https://travis-ci.org/WebAssembly/binaryen/jobs/272576776 I'm going to let this PR build go though and we'll see.

Reply:
It looks like it runs each yaml list item (starting with `-`) as a separate script, but this does the right thing.